ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/apparals/footwear/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description=""
+    jcr:primaryType="cq:Tag"
+    jcr:title="footwear"
```

Comment:
jcr:title="Footwear" , jcr:title for all tags to have First letter caps

Reply:
These changes are taken care